web_hidden_fields/README.rst

```diff
@@ -0,0 +1,82 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================
+Web Hidden Fields
+=================
+
+This module allow hide fields to all users or groups, or to a specific user or
+group, without change the views.
+
+Installation
+============
+
+Copy this module to your addons path and go to Applications and Install.
+
+Configuration
+=============
+
+Don't need any special configuration.
+
+Usage
+=====
+
+Go to Configuration -> Hidden Fields -> Hidden Fields.
```

Comment:
@acysos I think Technical -> Database streucture is a better place for this menu. Don't you think so?

![2018-01-29_23-28-31](https://user-images.githubusercontent.com/8736623/35537901-442ac9be-054c-11e8-867c-84b41fb7e624.png)

Reply:
Yes, It's good idea